### ISSUE TITLE
Crashes on MacOS (CRITICAL - normcap:152)

### ISSUE THREAD

**Issue by fedka** [dynobo/normcap#586]
### What happened?

Hi!

I've installed the latest version 0.5.3 from .dmg on macOS Big Sur 11.7.10, gave it access to screen recording. But when I run it, the initial screen just flashes, disappears and nothing happens. See the terminal output below.

### How did you install NormCap?

DMG installer (macOS)

### Operating System + Version?

macOS Big Sur 11.7.10

### Debug log output?*

```shell
2024-01-09 16:14:52.357 NormCap[1433:5157677] Configuring isolated Python...
2024-01-09 16:14:52.359 NormCap[1433:5157677] Pre-initializing Python runtime...
2024-01-09 16:14:52.361 NormCap[1433:5157677] PythonHome: /Applications/NormCap.app/Contents/Resources/support/python-stdlib
2024-01-09 16:14:52.368 NormCap[1433:5157677] PYTHONPATH:
2024-01-09 16:14:52.368 NormCap[1433:5157677] - /Applications/NormCap.app/Contents/Resources/support/python310.zip
2024-01-09 16:14:52.368 NormCap[1433:5157677] - /Applications/NormCap.app/Contents/Resources/support/python-stdlib
2024-01-09 16:14:52.368 NormCap[1433:5157677] - /Applications/NormCap.app/Contents/Resources/support/python-stdlib/lib-dynload
2024-01-09 16:14:52.368 NormCap[1433:5157677] - /Applications/NormCap.app/Contents/Resources/app_packages
2024-01-09 16:14:52.368 NormCap[1433:5157677] - /Applications/NormCap.app/Contents/Resources/app
2024-01-09 16:14:52.368 NormCap[1433:5157677] Configure argc/argv...
2024-01-09 16:14:52.368 NormCap[1433:5157677] Initializing Python runtime...
2024-01-09 16:14:52.449 NormCap[1433:5157677] Installing Python NSLog handler...
2024-01-09 16:14:52.543 NormCap[1433:5157677] Running app module: normcap
2024-01-09 16:14:52.563 NormCap[1433:5157677] ---------------------------------------------------------------------------
2024-01-09 16:14:53.368 NormCap[1433:5157677] 16:14:53 - INFO    - normcap:49 - Start NormCap v0.5.3
2024-01-09 16:14:53.513 NormCap[1433:5157677] 16:14:53 - DEBUG   - normcap.gui.tray:79 - System info:
2024-01-09 16:14:53.513 NormCap[1433:5157677] {'normcap_version': '0.5.3', 'python_version': '3.10.13', 'cli_args': '/Applications/NormCap.app/Contents/MacOS/NormCap -v debug', 'is_briefcase_package': True, 'is_flatpak_package': False, 'is_appimage_package': False, 'platform': 'darwin', 'desktop_environment': <DesktopEnvironment.OTHER: 0>, 'display_manager_is_wayland': False, 'pyside6_version': '6.6.1', 'qt_version': '6.6.1', 'qt_library_path': '/Applications/NormCap.app/Contents/Resources/app_packages/PySide6/Qt/plugins, /Applications/NormCap.app/Contents/MacOS', 'locale': 'DEFAULT', 'config_directory': PosixPath('/Users/fedka/.config/normcap'), 'resources_path': PosixPath('/Applications/NormCap.app/Contents/Resources/app/normcap/resources'), 'tesseract_path': PosixPath('/Applications/NormCap.app/Contents/Resources/app_packages/bin/tesseract'), 'tessdata_path': PosixPath('/Users/fedka/.config/normcap/tessdata'), 'envs': {'TESSDATA_PREFIX': None, 'LD_LIBRARY_PATH': None}, 'screens': [Screen(left=0, top=0, right=1279, bottom=799, device_pixel_ratio=2.0, index=0, screenshot=None)]}
2024-01-09 16:14:53.515 NormCap[1433:5157677] 16:14:53 - DEBUG   - normcap.gui.settings:162 - Skip update of non existing setting (show_introduction: None)
2024-01-09 16:14:53.516 NormCap[1433:5157677] 16:14:53 - DEBUG   - normcap.gui.settings:162 - Skip update of non existing setting (cli_mode: False)
2024-01-09 16:14:53.516 NormCap[1433:5157677] 16:14:53 - DEBUG   - normcap.gui.settings:162 - Skip update of non existing setting (background_mode: False)
2024-01-09 16:14:53.516 NormCap[1433:5157677] 16:14:53 - DEBUG   - normcap.gui.settings:162 - Skip update of non existing setting (clipboard_handler: None)
2024-01-09 16:14:53.518 NormCap[1433:5157677] 16:14:53 - DEBUG   - normcap.gui.tray:397 - Listen on local socket v0.5.3-normcap.
2024-01-09 16:14:53.943 NormCap[1433:5157677] 16:14:53 - DEBUG   - normcap.ocr.tesseract:24 - Executing '/Applications/NormCap.app/Contents/Resources/app_packages/bin/tesseract --list-langs --tessdata-dir /Users/fedka/.config/normcap/tessdata'
2024-01-09 16:14:53.990 NormCap[1433:5157677] 16:14:53 - CRITICAL - normcap:148 - Uncaught exception!
2024-01-09 16:14:53.990 NormCap[1433:5157677] Traceback (most recent call last):
2024-01-09 16:14:53.990 NormCap[1433:5157677]   File "/Applications/NormCap.app/Contents/Resources/app/normcap/gui/tray.py", line 377, in _delayed_init
2024-01-09 16:14:53.990 NormCap[1433:5157677]     self.installed_languages = ocr.tesseract.get_languages(
2024-01-09 16:14:53.990 NormCap[1433:5157677]   File "/Applications/NormCap.app/Contents/Resources/app/normcap/ocr/tesseract.py", line 52, in get_languages
2024-01-09 16:14:53.990 NormCap[1433:5157677]     output = _run_command(cmd_args=cmd_args)
2024-01-09 16:14:53.990 NormCap[1433:5157677]   File "/Applications/NormCap.app/Contents/Resources/app/normcap/ocr/tesseract.py", line 35, in _run_command
2024-01-09 16:14:53.990 NormCap[1433:5157677]     _raise_on_error(proc)
2024-01-09 16:14:53.990 NormCap[1433:5157677]   File "/Applications/NormCap.app/Contents/Resources/app/normcap/ocr/tesseract.py", line 18, in _raise_on_error
2024-01-09 16:14:53.990 NormCap[1433:5157677]     raise subprocess.CalledProcessError(
2024-01-09 16:14:53.990 NormCap[1433:5157677] subprocess.CalledProcessError: Command '['/Applications/NormCap.app/Contents/Resources/app_packages/bin/tesseract', '--list-langs', '--tessdata-dir', '/Users/fedka/.config/normcap/tessdata']' died with <Signals.SIGABRT: 6>.
2024-01-09 16:14:53.992 NormCap[1433:5157677] 16:14:53 - CRITICAL - normcap:151 - System info: {'normcap_version': '0.5.3', 'python_version': '3.10.13', 'cli_args': '/Applications/NormCap.app/Contents/MacOS/NormCap -v debug', 'is_briefcase_package': True, 'is_flatpak_package': False, 'is_appimage_package': False, 'platform': 'darwin', 'desktop_environment': <DesktopEnvironment.OTHER: 0>, 'display_manager_is_wayland': False, 'pyside6_version': '6.6.1', 'qt_version': '6.6.1', 'qt_library_path': '/Applications/NormCap.app/Contents/Resources/app_packages/PySide6/Qt/plugins, /Applications/NormCap.app/Contents/MacOS', 'locale': 'DEFAULT', 'config_directory': PosixPath('/Users/fedka/.config/normcap'), 'resources_path': PosixPath('/Applications/NormCap.app/Contents/Resources/app/normcap/resources'), 'tesseract_path': PosixPath('/Applications/NormCap.app/Contents/Resources/app_packages/bin/tesseract'), 'tessdata_path': PosixPath('/Users/fedka/.config/normcap/tessdata'), 'envs': {'TESSDATA_PREFIX': None, 'LD_LIBRARY_PATH': None}, 'screens': [Screen(left=0, top=0, right=1279, bottom=799, device_pixel_ratio=2.0, index=0, screenshot=None)]}
2024-01-09 16:14:53.992 NormCap[1433:5157677] 16:14:53 - CRITICAL - normcap:152 - Unfortunately, NormCap has to be terminated due to an unknown problem.
2024-01-09 16:14:53.992 NormCap[1433:5157677] Please help improve NormCap by reporting this error, including the output above, on
2024-01-09 16:14:53.992 NormCap[1433:5157677] https://github.com/dynobo/normcap/issues/new
2024-01-09 16:14:53.993 NormCap[1433:5157677] Thanks!
```

**Comment by dynobo** [comment]
Hi @fedka, thanks for reporting this! 
Seems like the bundled `tesseract` crashed.
May I ask, which processor you have? I think the command `sysctl -n machdep.cpu.brand_string` should print it, otherwise you can find it in "About this Mac".

**Comment by fedka** [comment]
Sure,

Quad-Core Intel Core i7-4850HQ CPU @ 2.30GHz

**Comment by dynobo** [comment]
Thanks, @fedka. This is weird, it _should_ work on Intel. 

However, I noticed, that the package is build on macOS 12, maybe this causes issues?

I've just built a package of NormCap 0.5.4 on macOS 11, could try [installing that one](https://github.com/dynobo/normcap/releases/tag/internal), report back if it works for you?

**Comment by fedka** [comment]
Hi @dynobo!

Looks like the link is wrong. It leads to old 2022 source code.

**Comment by dynobo** [comment]
> Looks like the link is wrong. It leads to old 2022 source code.

Sorry, my bad, forgot to publish. :see_no_evil: 

Here's the correct link: 
https://github.com/dynobo/normcap/releases/tag/debug-macos

**Comment by fedka** [comment]
Yes, it's ok now, starts properly.

But it doesn't work. See video below.

[Screen record](https://github.com/dynobo/normcap/assets/5571716/f162c080-46e3-4d9b-acf8-e15246694dae)

**Comment by dynobo** [comment]
Should be fixed by #634.

**Comment by dynobo** [comment]
Please re-open, if issue re-occurs.
